framework/src/modules/dynamic_rewards/module.ts

```diff
@@ -146,13 +146,13 @@ export class DynamicRewardModule extends BaseModule {
 			context.getMethodContext(),
 			context.header,
 		);
-		context.contextStore.set(CONTEXT_STORE_KEY_BLOCK_REWARD, defaultReward);
+		context.contextStore.set(CONTEXT_STORE_KEY_DYNAMIC_BLOCK_REWARD, defaultReward);
```

Comment:
same as reward module, i think reward and reduction should be stored, and used in the `afterTransactionsExecute`

Reply:
As shu said, it should be as reward module. Reward, reduction should be stored in context `beforeTransactionsExecute` and used in `afterTransactionExecute`. (https://github.com/LiskHQ/lips/blob/main/proposals/lip-0071.md#before-transactions-execution)

Reply:
ah yes, I now see it in the LIP.. but since it's not part of the changes in the [LIP PR](https://github.com/LiskHQ/lips/pull/266) it must have been already implemented or am I missing another LIP PR ?

Reply:
I think somehow it was missed in earlier update. Could you update in this PR?

Reply:
I updated the hooks of dynamic reward module to follow the LIP.